e164.js

```diff
@@ -5,648 +5,680 @@
-  "808": [ "ZZ", "International Shared Cost Service (ISCS)" ],
-  "81": [ "JP", "Japan" ],
-  "82": [ "KR", "South Korea" ],
-  "84": [ "VN", "Vietnam" ],
-  "850": [ "KP", "North Korea" ],
-  "852": [ "HK", "Hong Kong" ],
-  "853": [ "MO", "Macau" ],
-  "855": [ "KH", "Cambodia" ],
-  "856": [ "LA", "Laos" ],
-  "86": [ "CN", "China" ],
-  "870": [ "ZZ", "Inmarsat SNAC" ],
-  "878": [ "ZZ", "Universal Personal Telecommunications (UPT)" ],
-  "880": [ "BD", "Bangladesh" ],
-  "881": [ "ZZ", "Global Mobile Satellite System (GMSS)" ],
-  "8810": [ "ZZ", "ICO Global (Mobile Satellite Service)" ],
-  "8811": [ "ZZ", "ICO Global (Mobile Satellite Service)" ],
-  "8812": [ "ZZ", "Ellipso (Mobile Satellite service)" ],
-  "8813": [ "ZZ", "Ellipso (Mobile Satellite service)" ],
-  "8816": [ "ZZ", "Iridium (Mobile Satellite service)" ],
-  "8817": [ "ZZ", "Iridium (Mobile Satellite service)" ],
-  "8818": [ "ZZ", "Globalstar (Mobile Satellite Service)" ],
-  "8819": [ "ZZ", "Globalstar (Mobile Satellite Service)" ],
-  "88213": [ "ZZ", "EMSAT (Mobile Satellite service)" ],
-  "88216": [ "ZZ", "Thuraya (Mobile Satellite service)" ],
-  "886": [ "TW", "Taiwan" ],
-  "90": [ "TR", "Turkey" ],
-  "91": [ "IN", "India" ],
-  "92": [ "PK", "Pakistan" ],
-  "93": [ "AF", "Afghanistan" ],
-  "94": [ "LK", "Sri Lanka" ],
-  "95": [ "MM", "Burma" ],
-  "960": [ "MV", "Maldives" ],
-  "961": [ "LB", "Lebanon" ],
-  "962": [ "JO", "Jordan" ],
-  "963": [ "SY", "Syria" ],
-  "964": [ "IQ", "Iraq" ],
-  "965": [ "KW", "Kuwait" ],
-  "966": [ "SA", "Saudi Arabia" ],
-  "967": [ "YE", "Yemen" ],
-  "968": [ "OM", "Oman" ],
-  "970": [ "PS", "Palestinian territories" ],
-  "971": [ "AE", "United Arab Emirates" ],
-  "972": [ "IL", "Israel" ],
-  "973": [ "BH", "Bahrain" ],
-  "974": [ "QA", "Qatar" ],
-  "975": [ "BT", "Bhutan" ],
-  "976": [ "MN", "Mongolia" ],
-  "977": [ "NP", "Nepal" ],
-  "98": [ "IR", "Iran" ],
-  "992": [ "TJ", "Tajikistan" ],
-  "993": [ "TM", "Turkmenistan" ],
-  "994": [ "AZ", "Azerbaijan" ],
-  "995": [ "GE", "Georgia" ],
-  "99544": [ "AB", "Abkhazia" ],
-  "996": [ "KG", "Kyrgyzstan" ],
-  "998": [ "UZ", "Uzbekistan" ]
+	"1201": ["US","United States"],
+	"1202": ["US","United States"],
+	"1203": ["US","United States"],
+	"1204": ["CA","Canada"],
+	"1205": ["US","United States"],
+	"1206": ["US","United States"],
+	"1207": ["US","United States"],
+	"1208": ["US","United States"],
+	"1209": ["US","United States"],
+	"1210": ["US","United States"],
+	"1212": ["US","United States"],
+	"1213": ["US","United States"],
+	"1214": ["US","United States"],
+	"1215": ["US","United States"],
+	"1216": ["US","United States"],
+	"1217": ["US","United States"],
+	"1218": ["US","United States"],
+	"1219": ["US","United States"],
+	"1220": ["US","United States"],
+	"1223": ["US","United States"],
+	"1224": ["US","United States"],
+	"1225": ["US","United States"],
+	"1226": ["CA","Canada"],
+	"1227": ["US","United States"],
```

Comment:
Is this file also generated by your script? If so should we add a comment?

Reply:
Only a part of the file is generated, Ill add a comment.  I regex into the data structure and modify it inline.